skatepark/sass/blocks/_post-comments.scss

```diff
@@ -0,0 +1,169 @@
+@import "../../../blockbase/sass/blocks/button-mixins";
+
+.wp-block-post-comments {
+
+	line-height: var(--wp--custom--body--typography--line-height);
+
+	.reply {
+		margin-bottom: var(--wp--custom--margin--vertical);
+
+		a {
+			--wp--custom--button--typography--font-size: var(--wp--preset--font-size--normal);
```

Comment:
This is currently the wrong font size on this branch, and `-wp--preset--font-size--small` is correct. But I noticed these are about to be renamed, so I think 'normal' will be correct after that change.

Reply:
Yep, normal is the correct one, I just merged that branch so you can rebase to get that change